package/kedro_viz/server.py

```diff
@@ -21,7 +22,7 @@ def root():
 
 
 @app.route("/logs/nodes.json")
-def nodes():
+def nodes_old():
```

Comment:
Could we change the name to something like:
```suggestion
def nodes_deprecated():
```